template/java8/model/src/main/java/com/openfaas/model/Request.java

```diff
@@ -17,23 +21,45 @@
     private String queryRaw;
     private String pathRaw;
     private Map<String, String> path;
+    private InputStream inputStream;
 
-    public Request(String body, Map<String, String> headers) {
```

Comment:
Please keep this constructor also for backward compatibility.

Reply:
Yes, sorry, I have added it.